docs/source/user_guide/connector/config.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading configuration files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When ```DataPrep.Connector.connect()``` is called, the system reads from a configuration folder, which contains information about how to build connection with specific APIs.\n",
+    "\n",
+    "You can load the configuration files in the following two ways.\n",
+    "\n",
+    "### 1. Loading existing files from our github repo\n",
+    "\n",
+    "We maintain a github repo that contains configuration files for more than 20 websites.\n",
+    "\n",
+    "//todo: add reference\n",
```

Comment:
Is this meant to be done in the future or before this is merged?

Reply:
Yeah, since we are going to change the repo name, I am planning to point to the config repo here. And this should be done after the renaming process is done.